docs/source/host/invite-users.rst

```diff
@@ -13,39 +13,38 @@ Invite Your Friends
 
-.. note:: TIP: You can send other High Fidelity users clickable URLs that send them directly to your domain. Simply send them the URL ``https://hifi.place/<domain IP | place name>``. When they click the link, they will be prompted to open Interface.exe and land at the spawn point in your domain.
+.. note:: TIP: You can send other Vircadia users clickable URLs that send them directly to your domain. Simply send them the URL ``https://hifi.place/<domain IP | place name>``. When they click the link, they will be prompted to open Interface.exe and land at the spawn point in your domain.
 
 ----------------------------------------------
-Add Your Domain to the GoTo App
+Add Your Domain to the Explore App
 ----------------------------------------------
 
-The GoTo app is designed to advertise public domains across the metaverse.
+The Explore app is designed to advertise public domains across the metaverse.
 
-In order to be featured on High Fidelity's GoTo app, your domain must have a purchased place name and it must be open to logged in users or everyone. This means that, in your Domain Settings, under 'Standard Permissions', either the 'anonymous' or 'logged-in' group must have the Connect permission.
+In order to be featured on Vircadia's Explore app, you must add a location marker by opening the app and going to the spot in your domain that you want people to spawn at. In your app, scroll to the bottom and click the "Add Location" button, fill it out and you should be set! You may want to open your domain to the public, such as being open to logged in users or everyone. This means that, in your Domain Settings, under 'Standard Permissions', either the 'anonymous' or 'logged-in' group must have the Connect permission.
 
-To restrict access to your domain, you will need to create a blacklist of users without permission to connect to your domain. This can be set up by username, IP address, MAC address or machine fingerprint. We do not currently have a way to advertise your domain in the GoTo app, while restricting access to a specific guest list.
+To restrict access to your domain, you will need to create a blacklist of users without permission to connect to your domain. This can be set up by username, IP address, MAC address or machine fingerprint.
 
 ----------------------------------------------------------------
-Promote Your Community Event with High Fidelity
+Promote Your Community Event with Vircadia
 ----------------------------------------------------------------
 
-To get your event or experience added to the calendar on our website and added as a Feature Card in GoTo, you can submit it to us by filling out `this form <https://docs.google.com/forms/d/1KYTZCSdwX7oSW7Z7NoTVrIeWAor-6eAaQt072WSKWnc/edit>`_. 
+To get your event or experience added to the calendar on our website, you can submit it to us by filling out by joining our `Discord<https://discordapp.com/invite/Pvx2vke>` server and reaching out to a Core Maintainer.
```

Comment:
There was discussion about changing "Core Maintainers" to "Core Team" (because "maintainers" implies just maintenance work is being done.

Reply:
I've made the change.